charts/linkerd-control-plane/values.yaml

```diff
@@ -30,6 +30,8 @@ enableEndpointSlices: true
 # -- enables the use of pprof endpoints on control plane component's admin
 # servers
 enablePprof: false
+# -- enables the creation of pod disruption budgets for control plane components
+enablePodDisruptionBudget: false
```

Comment:
I think we need to add `deploymentStrategy` into the values file too so that it will show up in the chart readme

Reply:
Done.